### DISCUSSION TITLE
China Pull failed

### DISCUSSION THREAD

**Discussion by LingNine** [teslamate-org/teslamate#4000] [General]
I use a cloud server，The error information is as follows
ERROR: for teslamate  error pulling image configuration: download failed after attempts=6: dial tcp 108.160.162.102:443: i/o timeout
ERROR: error pulling image configuration: download failed after attempts=6: dial tcp 108.160.162.102:443: i/o timeout

**Comment by simonz0118** [comment]
I have the same problem. Have you solved it yet?

error pulling image configuration: download failed after attempts=6: dial tcp 108.160.166.9:443: i/o timeout

**Reply by LingNine** [reply]
未解决；
计划周末倒腾一下；
现在怀疑是被墙了，计划找手动更新的方法更新。

**Comment by JakobLichterfeld** [comment]
As the IP you provide is not docker hub your are not using TeslaMate
